src/librustc/ty/context.rs

```diff
@@ -1,3 +1,5 @@
+// ignore-tidy-filelength
```

Comment:
:(

Reply:
Haha I know... gets it compiling though. Not immediately obvious how to refactor. That's for another PR I think.